scheduled-jobs/build/ocp4_scan/Jenkinsfile

```diff
@@ -11,7 +11,7 @@ node {
     ] )
 
     def versionJobs = [:]
-    for ( version in commonlib.ocp4Versions ) {
+    for ( version in commonlib.ocp4Versions() ) {
```

Comment:
This needs to change since ocp4_scan / ocp4 is controlled by `freeze_automation` even for eol releases we need to build